src/app/middleware/localStorageMiddleware.ts

```diff
@@ -20,16 +21,35 @@ const localStorageMiddleware = (store: any) => (next: any) => async (action: App
       break;
 
     case RESOURCEPATHS_ADD_SUCCESS: {
-      const navigationEntry = performance.getEntriesByType('navigation')[0] as PerformanceNavigationTiming;
-      if(navigationEntry && navigationEntry.loadEventEnd > 0) {
-        await saveResourcesCollection(action.response);
+      const collections = await collectionsCache.read();
+      if (collections && collections.length > 0) {
+        const item = collections[0];
```

Comment:
Valid question. We can use the isDefault flag to check this here. In the future, we will just be changing the default collection so that the user doesn't need to select a collection each time